backend/Lingva.WebAPI/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "LingvaConnection": "Server=SERGEY-DELL\\SQLEXPRESS;Database=Lingva;Trusted_Connection=True;"
+    "LingvaConnection": "Server=(localdb)\\mssqllocaldb;Database=Lingva;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
:-) Try to make sensitive variables as environment template variables. And fill it in code.
Because everybody will be getting in stuck due this )